x-pack/test_serverless/functional/test_suites/common/discover/group2/_data_grid_doc_navigation.ts

```diff
@@ -24,7 +24,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await security.testUser.setRoles(['kibana_admin', 'test_logstash_reader']);
       await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/logstash_functional');
       await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover');
-      await PageObjects.svlCommonPage.loginWithRole('viewer');
+      await PageObjects.svlCommonPage.loginWithPrivilegedRole();
```

Comment:
Any reason to change a SAML role here? I thought PR is only related to migration from basic auth.

Reply:
https://github.com/elastic/kibana/pull/188128/commits/5e81f8626c28215c12b60c0d9d3cd771f37cb7da

Reply:
@wayneseymour Would it be possible to also update the following files?

x-pack/test_serverless/functional/test_suites/common/context/_context_navigation.ts
x-pack/test_serverless/functional/test_suites/common/context/_filters.ts
x-pack/test_serverless/functional/test_suites/common/context/_size.ts
x-pack/test_serverless/functional/test_suites/common/discover/group5/_url_state.ts

Reply:
https://github.com/elastic/kibana/pull/188128/commits/85ccae1162febf5c608dde67979537ecdd5a935f